modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -340,6 +342,18 @@ private void _configureTaskBuildClientExtension(
 			createClientExtensionConfigTaskProvider,
 		TaskProvider<Zip> zipTaskProvider) {
 
+		createClientExtensionConfigTaskProvider.configure(
+			new Action<CreateClientExtensionConfigTask>() {
+
+				@Override
+				public void execute(CreateClientExtensionConfigTask task) {
+					TaskInputsInternal inputs = task.getInputs();
+
+					inputs.file(project.file("client-extension.yaml"));
```

Comment:
Lets use the `_CLIENT_EXTENSION_YAML` variable instead of hard coding the `client-extension.yaml` name.

Reply:
ok